.github/workflows/release-docker-image.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release TCPORTAL Image
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish Pre-release' || format('Release {0}', inputs.release_type)}} Docker Image by @${{ github.actor }}  
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: 'The type of release'
+        options:
+        - Major
+        - Minor
+        - Patch
+        - Snapshot
+  pull_request:
+
+jobs:
+  bumpVersion:
```

Comment:
Suggest we call this incrementVersionNumber - bump implies it is only for smaller releases